src/components/Footer/Footer.tsx

```diff
@@ -23,87 +24,85 @@ interface FooterProps {
  * as navigational NYPL.org links, social media links, copyright, and
  * NYPL building facade image.
  */
-export const Footer = chakra(
```

Comment:
Darn, is `chakra` another export we need? It seems to work without this wrapper but I'll include it a release soon.

Reply:
Ticket: https://jira.nypl.org/browse/DSD-1735

Reply:
I was confused by this but everything seems to work the same without it (unlike useMediaQuery)